hearts/hearts.py

```diff
@@ -13,11 +13,79 @@ def __hash__(self):
 
 
 class Game(object):
-    def __init___(self, players, points_to_win=100):
-        self.points_to_win = points_to_win
+    def __init__(self, players, points_to_win=100):
+        self.max_points = points_to_win
         self.players = players
-        self.round_counter = 1
+        self.rounds = []
+        self.round_number = 1
+
         shuffle(self.players)
+        self.create_round()
+
+    def create_round(self):
+        pass_direction = {0: 'keep', 1: 'left', 2: 'right', 3: 'across'}
```

Comment:
Have to update this to accommodate for starting indexing at zero (and might as well just make it a list `['left', 'right', 'across', 'keep']`)